index.html

```diff
@@ -223,63 +252,59 @@ <h3 class="contact__text--header">Contact Us</h3>
           >
             <input
               type="text"
-              name="name"
-              id="name"
-              placeholder="Text Input"
+              name="full_name"
+              id="full_name"
+              placeholder="Full Name"
               required
               maxlength="30"
             />
             <input
               type="email"
               name="email"
               id="email"
-              placeholder="jeremy@gmail.com"
+              placeholder="Email Address"
               required
             />
             <textarea
               name="message"
               id="message"
-              cols="30"
-              rows="10"
+              rows="6"
               placeholder="Enter text here"
               required
               maxlength="500"
             ></textarea>
             <input class="btn" type="submit" value="See Projects" />
```

Comment:
- [ ] Kindly follow the Figma design for the desktop version of your form
<img width="866" alt="Screen Shot 2022-04-28 at 11 58 25" src="https://user-images.githubusercontent.com/28307960/165728566-66c647c6-e292-49d3-bc7d-e9678e94cd50.png">
The full name should be replace by first and last name, instead of having full name